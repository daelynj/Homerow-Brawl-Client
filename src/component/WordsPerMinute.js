import React from 'react';

function WordsPerMinute(props) {
  const wpm = "WPM: " + Math.round(props.textLength / (props.finishTime / 60))

  return <div className="wpm">{wpm}</div>;
}

export default WordsPerMinute;
