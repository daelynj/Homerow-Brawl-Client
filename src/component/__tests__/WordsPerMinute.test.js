import React from 'react';
import WordsPerMinute from '../WordsPerMinute';
import renderer from 'react-test-renderer';

describe("WordsPerMinute", () => {
  it("renders the expected words per minute", () => {
    const tree = renderer.create(
      <WordsPerMinute
        wordsLength = {60}
        finishTime = {60}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
