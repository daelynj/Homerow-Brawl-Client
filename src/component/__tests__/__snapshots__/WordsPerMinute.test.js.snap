// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordsPerMinute renders the expected words per minute 1`] = `
<div
  className="wpm"
>
  WPM: 60
</div>
`;
